src/main/java/at/ac/tuwien/kr/alpha/config/CommandLineParser.java

```diff
@@ -0,0 +1,301 @@
+package at.ac.tuwien.kr.alpha.config;
```

Comment:
Since lots of content in this file have been copied from Main.java, the license header from the original file should also be copied. (This may also be the case for other files.)

Reply:
done